app/contributing/markdown-rules.md

```diff
@@ -78,13 +83,13 @@ domain (`docs.konghq.com`).
     For example, if the final link for a page will be `docs.konghq.com/konnect/servicehub`,
     you would write `/konnect/servicehub`.
```

Comment:
I have no idea why Vale is triggering on content inside backticks here, or why it triggered on `{{site.konnect_short_name}}`.